faq.md

```diff
@@ -0,0 +1,157 @@
+
+<br/>
+
+Q: How often will data models be released?
+
+A: Approximately once or twice a year, concurrent with the needs of data releases. 
+
+<br/>
+
+Q: How will spatial information be stored?
+
+A: Working groups are currently working on trying to define standards, please contact [HTAN Working Groups](https://www.synapse.org/#!Synapse:syn17022193/wiki/585904) if you have any comments or concerns. 
+
+<br/>
+
+Q: Can we have an intermediate data storage bucket? 
+
+A: Please contact your [DCC liaison](dcc-liaison) for consideration on a case by case basis. 
+
+<br/>
+
+Q: How can we use docker images/workflows/reproducible research practices? 
+
+A: You can use Synapse's internal docker registry, or you can link to a docker registry of your preference. Additionally, we will provide tools for linking your workflow to data you have access to on Synapse.
+
+<br/>
+
+## Data Uploader
+
+Q: How do I upload a \<dataset type> dataset?
+
+A: Please follow the standard operating procedures for data upload: [Data Ingress SOP](https://github.com/Sage-Bionetworks/HTAN-data-ingress-documentation-draft/blob/master/DOCUMENTATION.md).
+
+<br/>
+
+Q: What format does my data need to be in for submission? E.g do I need to organize my \<dataset type> dataset in a specific file/folder hierarchy?
+
+A: The topmost level folder needs to be descriptive because it will be used in the Data Ingress Service. Your data can be in any format or hierarchy under that folder. 
+
+<br/>
+
+Q: What annotations/metadata should we include for a specific data type?
+
+A: When you specify the dataset type (e.g. scRNASeq) to the programmatic client or the web based application, you will be provided with a spreadsheet and/or a web interface to fill in the required metadata/annotations. We are currently coordinating with the [HTAN Working Groups](https://www.synapse.org/#!Synapse:syn17022193/wiki/585904), as we prepare the relevant data models. 
+
+<br/>
+
+Q:  How can I update datasets and related metadata?
+
+A:  You can use the programmatic or web interface to add files to a dataset, change metadata associated with files in a dataset, and mark files as obsolete. See the “Update existing metadata” section of the Data Ingress SOP. 
+
+<br/>
+
+Q: How can I give metadata  update access to someone else in my institute?
+
+A: Contact your team manager to add users to your center’s Synapse Team.  
+
+<br/>
+
+Q: We have terabytes of imaging data. How long of an upload time should we expect? 
+
+A: This depends on your internet bandwidth, where your data is located, your internet network, and more. If you are experiencing slow uploads or data bottlenecks, your DCC liaison is available to help optimize your specific use case.
+
+<br/>
+
+Q: How should I keep track of my tissue samples and associated data? Is there an identifier schema I need to follow?
+
+A: Please follow the sample identifier standard operating procedure: [HTAN Identifiers SOP](https://bit.ly/htan_ids).
```

Comment:
might want to share examples of how centers are using identifiers and "blocks" etc

Reply:
Added in more info from the HTAN identifiers SOP and RFC but it might be a bit too wordy for an FAQ...